### PR TITLE
[DNM] log for debug

### PR DESCRIPTION
**Pull request by ekexium**
### What is changed and how it works?

Issue Number: Close #xxx

What's Changed:

```commit-message
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

```release-note
Please add a release note.

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with None.
If this PR will be picked to release branch, then a release note is probably required.
```

### CODE REVIEW

**Comment by ekexium** [comment]
It might not be useful.
Just FYI @you06

File: components/raftstore/src/store/fsm/apply.rs [hunk 1]
```diff
@@ -1791,6 +1792,7 @@ where
     fn handle_put(&mut self, ctx: &mut ApplyContext<EK>, req: &Request) -> Result<()> {
         PEER_WRITE_CMD_COUNTER.put.inc();
         let (key, value) = (req.get_put().get_key(), req.get_put().get_value());
+        debug!("handle put"; "key" => log_wrappers::hex_encode_upper(key), "value" => log_wrappers::hex_encode_upper(key), "index" => ctx.exec_log_index);
```
Comment (you06) [inline components/raftstore/src/store/fsm/apply.rs:1795]:
```suggestion
        debug!("handle put"; "key" => log_wrappers::hex_encode_upper(key), "value" => log_wrappers::hex_encode_upper(value), "index" => ctx.exec_log_index);
```

**Pull request closed without merging**